scripts/kibana-post-start.sh

```diff
@@ -8,3 +8,6 @@ do
    /usr/bin/sleep 1
 done
 echo "[INFO] Kibana is reachable and returned a 200 status" >> $KIBANA_STARTUP_LOG
+
+# Call script to clean up old .kibana_* indices
+python /usr/local/kibana-7.5.2-linux-x64/scripts/rmOldKibanaIndices.py
```

Comment:
A third ExecStartPost in the kibana.service file would be a better place to run this script from (IMO).